data/presets/highway/cycleway/crossing/bicycle_foot.json

```diff
@@ -4,8 +4,10 @@
             "fr",
             "lt",
             "pl",
+            "de",
```

Comment:
Good catch! I created https://github.com/openstreetmap/id-tagging-schema/pull/1204 to improve this situation

Reply:
Great. Does that mean there's progress on this PR as well?